includes/core/server/request/class-get-reporting-payment-activity.md

```diff
@@ -13,26 +13,31 @@ The `WCPay\Core\Server\Request\Get_Reporting_Payment_Activity` class is used to
 | `date_start`| `set_date_start( string $date_start )`    |    No     |    Yes   |       -       |
 | `date_end`  | `set_date_end( string $date_end )`        |    No     |    Yes   |       -       |
 | `timezone`  | `set_timezone( string $timezone )`        |    No     |    Yes   |       -       |
+| `currency`  | `set_currency( string $currency )`        |    No     |    Yes   |       -       |
 
 The `date_start` and `date_end` parameters should be in the 'YYYY-MM-DDT00:00:00' format.
 The `timezone` parameter can be passed as an offset or as a [timezone name](https://www.php.net/manual/en/timezones.php).
+`currency` should be a currency code for a store deposit currency.
 
 ## Filter
 
 When using this request, provide the following filter and arguments:
 
 - Name: `wcpay_get_payment_activity`
+- ? what
```

Comment:
There are no arguments for this hook. https://github.com/Automattic/woocommerce-payments/blob/develop/includes/core/server/README.md#providing-filters

Reply:
Reading this again I think I understand. I can see now that this documents essentially the name/endpoint of the API – aka the `hook` under the hood (inside WordPress, inside WooPayments server). 

My comments here are me musing about how we can make our API docs super clear, and focus on the most important info. The docs here explain how this PHP class is used to construct requests, which feels close to boilerplate code IMO – we'd be better off documenting the REST API.